messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -1873,7 +1878,7 @@ class MessagingTest : BaseMessagingTest() {
 internal suspend fun <T : Any> Messaging.waitFor(
     path: String,
     testCase: String,
-    duration: Duration = 10.seconds,
+    duration: Duration = 20.seconds,
```

Comment:
Sometimes the provisional contacts test failed because the hello handshake didn't complete in time. This bump in the timeout fixed that.